docs/reference/announcements/870.md

```diff
@@ -106,92 +37,22 @@ The Web Modeler **Deploy diagram** modal has changed, and clusters must now be p
 
 ### Helm charts
```

Comment:
@akeller let's also explicitly confirm with @aabouzaid whether this is still relevant for 8.7.0 or not please

Reply:
I've tagged him in a comment above on this PR - https://github.com/camunda/camunda-docs/pull/4890#issuecomment-2608434286

Reply:
If you want to merge quickly, I can remove this for now @MaxTru. Please let me know.

Reply:
Clarified and updated.